src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1760,6 +1761,16 @@ void PolicyHandler::OnCertificateUpdated(const std::string& certificate_data) {
 }
 #endif  // EXTERNAL_PROPRIETARY_MODE
 
+void PolicyHandler::OnPTUFinished(const bool ptu_result) {
```

Comment:
@tkireva It would be nice if you apply a bit different implementation like OnEmptyCertificateArrived.